spiceaidocs/docs/components/data-connectors/postgres/index.md

```diff
@@ -16,31 +16,100 @@ datasets:
 
 :::warning[Limitations]
 
-- The Postgres federated queries may result in unexpected result types due to the difference in DataFusion and Postgres size increase rules. Please explicitly specify the expected output type of aggregation functions when writing query involving Postgres table in Spice. For example, rewrite `SUM(int_col)` into `CAST (SUM(int_col) as BIGINT`.
+The Postgres federated queries may result in unexpected result types due to the difference in DataFusion and Postgres size increase rules. Please explicitly specify the expected output type of aggregation functions when writing query involving Postgres table in Spice. For example, rewrite `SUM(int_col)` into `CAST (SUM(int_col) as BIGINT`.
```

Comment:
This doesn't feel important enough to be a warning at the top of the docs page. Its more like a hint on how to use postgres effectively in Spice, I would mark it as info and move it down the page.

Reply:
Moved it to the `Types` section as an info